$NetBSD$

Permit unicorn-2.1.3.

This can be problematic if used for MIPS, please see
<https://github.com/Gallopsled/pwntools/issues/2560> and
<https://github.com/unicorn-engine/unicorn/issues/2134>.

This will be fixed in Unicorn 2.1.4.

--- pyproject.toml.orig	2025-05-05 14:43:02.271778077 +0000
+++ pyproject.toml
@@ -50,7 +50,7 @@ dependencies = [
     "psutil>=3.3.0",
     "intervaltree>=3.0",
     "sortedcontainers",
-    "unicorn>=2.0.1, !=2.1.3",
+    "unicorn>=2.0.1",
     "six>=1.12.0",
     "rpyc",
     "colored_traceback<0.4; python_version < '3'",
